### ISSUE TITLE
`wsl --help` text refers to `--shut-down`, when the correct argument is `--shutdown`

### ISSUE THREAD

**Issue by me-and** [microsoft/WSL#8900]
### Version

Windows version: 10.0.22622.601

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Repro Steps

Run `wsl --help` from a PowerShell environment with the Windows store version of WSL (WSL version: 0.67.6.0)

The help text includes the following:

```
    --shut-down
        Immediately terminates all running distributions and the WSL 2
        lightweight utility virtual machine.
```

However, running `wsl --shut-down` produces the following:

```
PS C:\Windows\System32> wsl --shut-down
Invalid command line argument: --shut-down
Please use 'wsl.exe --help' to get a list of supported arguments.
```

### Expected Behavior

Either the output from `wsl --help` gives the argument as `--shutdown`, or the command `wsl --shut-down` works.

Ideally `--shut-down` would be an alias for `--shutdown`, so both versions work.

### Actual Behavior

PS C:\Windows\System32> wsl --shut-down
Invalid command line argument: --shut-down
Please use 'wsl.exe --help' to get a list of supported arguments.

**Comment by Biswa96** [comment]
This sounds like a typo.

**Comment by cerebrate** [comment]
One question that occurs to me is the source of _wsl.exe_; I don't see this issue (my help message says `--shutdown`), and my versions are identical apart from the Windows build:

```
 ❯ wsl --version
WSL version: 0.67.6.0
Kernel version: 5.15.62.1
WSLg version: 1.0.44
MSRDC version: 1.2.3401
Direct3D version: 1.606.4
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22000.978
```

**Comment by NotTheDr01ds** [comment]
@cerebrate Agreed, I can't repro either, but OP is on Insider (`10.0.22622.601`), while you and I are on GA Win 11.  So I'm wondering if that means that the package in the Store install (available only to insiders) is different than the one that is installed from the `.msix` that we installed.  That would be odd, but I can't rule it out.

```
❯ wsl.exe --help | iconv -f utf16 -t utf8 | grep -A 2 shut | clip.exe
    --shutdown
        Immediately terminates all running distributions and the WSL 2
        lightweight utility virtual machine.

❯ wsl.exe --version | clip.exe

WSL version: 0.67.6.0
Kernel version: 5.15.62.1
WSLg version: 1.0.44
MSRDC version: 1.2.3401
Direct3D version: 1.606.4
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.22000.978
```

Since `.601` is an Insider Build,

**Comment by me-and** [comment]
I have no idea whether it's relevant, but it seems plausible: I'm using British English / en_GB, and "shutdown" vs "shut-down" is the sort of thing that an overzealous US-to-UK English convertor could well have broken.

I did note that the non-Store version of WSL doesn't have this error, but the only variants I've tested with are the built-in non-Store version and the Store version that installed automatically.

**Comment by L2-L2** [comment]
I can confirm the shut-down issue, even though I'm using US English. Initially I thought it to be a typo, but it seems a matter of automatic translation.

**Comment by NotTheDr01ds** [comment]
@L2-L2  Are you on an Insider build as well?

**Comment by L2-L2** [comment]
No, just an average user, having installed WSL2 and trying to figure out
why I can't shut it down.

**Comment by NotTheDr01ds** [comment]
@L2-L2 Interesting.  I'm still not seeing the problem on any of my systems (1.0.0 on Windows 10, 1.0.0 on Windows 11, or an older Windows 10 system that hasn't yet been updated).
